app/assets/stylesheets/contact.scss

```diff
@@ -0,0 +1,61 @@
+div.contacts {
+  width: 460px;
+  margin: 100px auto;
+  color: #2e405c;
+  font-size: 1.1rem;
+  background-color: #fbfbfb;
+  span.help-block {
+    padding-left: 103px;
+    color: red;
+    font-weight: 300;
+  }
+  div#status-area {
+    display: none;
+  }
+  form#new_contact {
+    position: relative;
+    margin-bottom: 100px;
+
+  }
+  #new_contact div:nth-child(2) {
```

Comment:
remove this div:nth-child(2) this thing. It's difficult to support it. what if you add a new div SO YOU MUST change CSS.